internal/venice-common/src/main/java/com/linkedin/venice/throttle/EventThrottler.java

```diff
@@ -209,19 +212,22 @@ public void onExceedQuota(Time time, String throttlerName, long currentRate, lon
       if (quota == 0) {
         sleepTimeMs = timeWindowMS;
       } else {
-        sleepTimeMs = Math.round(excessRate / quota * Time.MS_PER_SECOND);
```

Comment:
github won't let me comment on lines you didn't touch, but why make excessRate a double?  A long minus another long can't possibly result in a value that needs a double.  I'm wondering if there's some widening gotcha that's resulting in the funny result.

Reply:
its making it double because of following division operation. ie. 
`excessRate / quota * Time.MS_PER_SECOND`

We can transform it into a (double) excessRate/quota later.. but basically the same thing